### DISCUSSION TITLE
UGS WILL NOT START UP.

### DISCUSSION THREAD

**Discussion by undertheknife2022** [winder/Universal-G-Code-Sender#2549] [Help] [answered]
Latest version of UGS downloaded 2.1.7, Java 1.8.0_411-b09.
UGS will not OPEN. Please Help, can't find a solution, read all threads I could find, tried everything they suggested.
Thanks in advance

**Comment by breiler** [comment, marked as answer]
Java is included in the package, so installing Java separately is not needed.

Try and delete this directory and start again:
C:/Users/[your username>]/AppData/Roaming/ugsplatform

If that doesn't help I need your log file from:
C:/Users/[your username>]/AppData/Roaming/ugsplatform/var/log/messages.log

**Comment by undertheknife2022** [comment]
Hello breiler,
Deleting the file C:/Users/[your username>]/AppData/Roaming/ugsplatform and restarting was the cure!! I've been hunting for this answer for many days and I very much appreciate your assistance!!
May you be blessed beyond your imagination!!
Thank you!!
G.

**Comment by khturnings** [comment]
The number of times I have seen this instruction to delete this file would it be an option to add a help menu item that provides a list of common fixes like this to be used before raising discussions or issues on Git
